frontend/src/cards/NewTaskCard.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {ChangeEvent, useEffect, useState} from 'react';
+import './NewTaskCard.css';
+import {useNavigate} from 'react-router-dom';
+import axios from 'axios';
+import useTasks from '../hook/useTasks';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import {TaskModel} from "../model/TaskModel";
+import {format} from 'date-fns';
+
+function NewTaskCard() {
+    const [inputTaskName, setInputTaskName] = useState('');
+    const [inputCreator, setInputCreator] = useState('');
+    const [inputCategory, setInputCategory] = useState('');
+    const [createdate, setCreatedate] = useState<string>(getTodayDate());
+    const [selectedDate, setSelectedDate] = useState<Date | null>(null);
+    const [inputDescription, setInputDescription] = useState('');
+    const [inputAmoundOfPeople, setInputAmoundOfPeople] = useState<number | null>(null);
+    const navigate = useNavigate();
+    const {getAllTasks} = useTasks();
+    const handleDateChange = (date: Date | null) => {
+        setSelectedDate(date);
+    };
+
+    useEffect(() => {
+        setCreatedate(getTodayDate());
+    }, []);
+
+    function useTextInput(event: ChangeEvent<HTMLTextAreaElement>) {
```

Comment:
this function could be renamed to "handleInputChange" or something similar to better reflect its purpose and to prevent risk of confusion with react hooks that have 'use' as a prefix